mobile/src/main/java/ch/epfl/sdp/mobile/infrastructure/persistence/store/Query.kt

```diff
@@ -40,6 +40,17 @@ interface Query {
    */
   fun whereNotEquals(field: String, value: Any?): Query
 
+  /**
+   * Filters the results by keeping only the documents which contain a string value prefixed with
+   * the given [prefix] for the given [field].
+   *
+   * @param field the field for which the difference is checked.
+   * @param prefix the prefix that the value should start with.
+   *
+   * @return the updated [Query].
+   */
+  fun startsWith(field: String, prefix: String): Query
```

Comment:
This doesn't match the Firestore API. I think what you should do is `fun whereGreaterThan(field: String, value: Any?): Query`.

Reply:
I think we need to write that this file follow the Firestore API somewhere (In the interface comment ?)